docs/jps_guide/titlepage.tex.in

```diff
@@ -0,0 +1,29 @@
+%% this file is generated automatically. All manual changes will be deleted.
```

Comment:
With which script do I create this file? So I understand that this is a template. Can you add which script is used to create this template?

Reply:
see the description of the PR.

Reply:
I've overlooked this. Could we also write this description in the file? As I understand it, this will help at some point in time.